test/dhtest.c

```diff
@@ -486,13 +492,13 @@ static int run_rfc5114_tests(void)
 {
     int i;
     for (i = 0; i < (int)OSSL_NELEM(rfctd); i++) {
-        DH *dhA, *dhB;
+        DH *dhA = NULL, *dhB = NULL;
         unsigned char *Z1 = NULL, *Z2 = NULL;
```

Comment:
yes, those 4 para should be defined out of the for loop, code has been modified:)